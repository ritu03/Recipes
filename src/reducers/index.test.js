// import rootReducer from './index'
// import { createStore } from 'redux'

// describe('rootReducer', () => {
//   test('rootReducer invoked without crashing', () => {
//     expect(rootReducer).toMatchSnapshot()
//   })
//   test('rootReducer invoked for CLEAR_STORE action', () => {
//     const store = createStore(rootReducer, [])
//     store.dispatch({ type: 'CLEAR_STORE' })
//     const actual = store.getState().menuItems
//     const expected = { data: null, error: false }
//     expect(actual).toEqual(expected)
//   })
// })
