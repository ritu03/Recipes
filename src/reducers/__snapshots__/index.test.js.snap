// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer rootReducer invoked without crashing 1`] = `[Function]`;
